src/facade/dragonfly_connection.cc

```diff
@@ -159,8 +159,9 @@ void Connection::PipelineMessage::SetArgs(const RespVec& args) {
     size_t s = buf.size();
     if (s)
       memcpy(next, buf.data(), s);
+    next[s] = '\0';
     this->args[i] = MutableSlice(next, s);
-    next += s;
+    next += (s + 1);
   }
```

Comment:
I don't understand. Do you rely now on the \0 to be present? But what about regular sync dispatch? What about StoredCmd (that is tightly packed)?

Reply:
I do not strictly rely on delimiter but I do want that the arguments have space between them to put '\0' delimiters when needed. The reason for this is what lua requires '\0' delimiters so I am planning to set them when passing the arguments to lua. out of scope of this PR.